azure-pipelines.yml

```diff
@@ -87,18 +87,18 @@ jobs:
     # -deps-pre : test pre-release versions of (python) dependencies)
     #
     # 64 bit - HDF5 1.10
-      py39-hdf5110:
-        python.version: '3.9'
-        TOXENV: py39-test-deps,py39-test-mindeps,py39-test-deps-pre
+      py310-hdf5110:
```

Comment:
We were roughly taking a diagonal across the matrix of possible dependencies - testing a few different Python versions and HDF5 versions at the same time. Now they're getting all bunched up on the same Python version. We might want to shuffle some along to newer Python versions - or we could decide it's redundant with the wheel builds on Github actions and cut some of them out. :shrug:

Reply:
your call ! I wasn't aware this was the strategy, but I think I'd be comfortable with dropping some of these.